app/models/order.rb

```diff
@@ -2,4 +2,22 @@
 
 class Order < ApplicationRecord
   has_many :line_items, dependent: :destroy
+
+  validates :name, :address, :email, :pay_type, presence: true
+
+  def add_line_items_from_cart(cart)
```

Comment:
Why do you need order_price in this model? Can you just calculate the sum from line_items without saving it to DB